### ISSUE TITLE
`no-single-composition` rule

### ISSUE THREAD

**Issue by jfmengels** [jfmengels/eslint-plugin-lodash-fp#8]
Require composition methods to have at least two functions passed to it.
(Might as well check that it doesn't have 0)

### Fail

```js
_.flow(fn);
```

### Good

```js
_.flow(fn1, fn2);
```
